app/lib/actions/katello/repository/discover.rb

```diff
@@ -1,8 +1,9 @@
 module Actions
   module Katello
     module Repository
-      class Discover < Actions::Base
+      class Discover < Actions::EntryAction
```

Comment:
What'd this do?

Reply:
This is a class that extends Actions::Base, and really has no affect here, but in general any action we call from controllers should extend it instead of Base.  It allows for things like locking and has a few other features